lib/ProductOpener/Display.pm

```diff
@@ -138,6 +139,7 @@ use Apache2::Const ();
 $memd = new Cache::Memcached::Fast {
 	'servers' => [ "127.0.0.1:11211" ],
 	'utf8' => 1,
+	'debug' => 1,
```

Comment:
unsupported? https://metacpan.org/pod/Cache::Memcached::Fast#debug

Reply:
right, i'll remove it

Reply:
Fixed in 0291c241e17fc1b682529b817da55e90414f5642